M/MPFI/build_tarballs.jl

```diff
@@ -37,9 +39,10 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("GMP_jll", v"6.2.0"),
-    Dependency("MPFR_jll", v"4.1.1"),
+    Dependency("GMP_jll"; compat="6.2.1"),
+    Dependency("MPFR_jll"; compat="4.2.0"),
```

Comment:
Doesn't v4.1.1 work?

Reply:
I went with the versions used by Julia 1.10. MPFR 4.1.1 doesn't support riscv.